openmdao/visualization/opt_report/tests/test_opt_report.py

```diff
@@ -592,7 +592,7 @@ def test_opt_report_mpi(self):
 
         prob.set_solver_print(level=0)
 
-        prob.driver = om.ScipyOptimizeDriver(optimizer='SLSQP', tol=1e-6, disp=False)
+        prob.driver = om.ScipyOptimizeDriver(optimizer='SLSQP', tol=1e-5, disp=False)
```

Comment:
This was done to reduce the number if iterations? Looks like this tests' purpose is just to make sure opt_report runs without raising an error anyway.